__tests__/cities.test.js

```diff
@@ -0,0 +1,60 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const request = require('supertest');
+const app = require('../lib/app');
+
+describe('city routes', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+
+  it('/cities should return a list of cities', async () => {
+    const resp = await request(app).get('/cities');
+    expect(resp.status).toEqual(200);
```

Comment:
i would add in an extra check here to make sure the endpoint is returning the data you want. For example check the length of the response:
```js
expect(resp.body.length).toBe(3)
```